main.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"math"
 
+	"gonum.org/v1/gonum/floats"
```

Comment:
This package is a *must*, thanks!

Reply:
:)

Reply:
another one is in the works for `complex128`:
- https://github.com/gonum/gonum/pull/1290